examples/release_quickstart_image.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
we can move this to ./scripts

Reply:
Good idea. Moved and updated the docker script.